source/MRViewer/MRMeshBoundarySelectionWidget.cpp

```diff
@@ -25,6 +26,10 @@ void BoundarySelectionWidget::enable( bool isEnaled )
     isSelectorActive_ = isEnaled;
     if ( !isSelectorActive_ )
     {
+        for ( auto& [obj, signal] : onMeshChangedSignals_ )
+            signal.disconnect();
+
+        onMeshChangedSignals_.clear();
```

Comment:
scoped connections should disconnect on clear, no need to call manualy

Reply:
ok, Thanks.  will remove it